test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/BaseCcmRule.java

```diff
@@ -232,4 +232,8 @@ public ProtocolVersion getHighestProtocolVersion() {
       return DefaultProtocolVersion.V3;
     }
   }
+
+  public CcmBridge getCcmBridge() {
+    return ccmBridge;
+  }
```

Comment:
Why is this method moved here from CustomCcmRule?

Reply:
Could be an accident, but this was so long ago that I need to look at it again.

Reply:
I think it was to allow combining new rule with all other rules by passing ccmBridge as an argument.